wordhelper/crossword_clue.py

```diff
@@ -0,0 +1,19 @@
+import requests
+import urllib.parse
+from bs4 import BeautifulSoup
+
+def crossword_clue(clue, pattern = ''):
```

Comment:
Would you mind adding some tests to this? That way at least we'll know the `bs4` module install is working on a different machine.

Reply:
I've added a couple tests but are currently skipping them due to build machines not being able to to talk to the internet. Do you have any clues on whether it is possible for the build machines to do so or is there something I can use to determine it's the build machine so that we can conditionally skip?